db_store.py

```
@@ -407,13 +407,23 @@ def defrag_availability(db_conn=None):
+            if table == "crew_availability":
+                select_query = "SELECT id, crew_id, start_time, end_time FROM crew_availability ORDER BY crew_id, start_time"
+                update_query = "UPDATE crew_availability SET end_time = ? WHERE id = ?"
+                delete_query = "DELETE FROM crew_availability WHERE id = ?"
+                id_col_idx = 1
+            else:
+                select_query = "SELECT id, appliance_id, start_time, end_time FROM appliance_availability ORDER BY appliance_id, start_time"
+                update_query = (
+                    "UPDATE appliance_availability SET end_time = ? WHERE id = ?"
+                )
+                delete_query = "DELETE FROM appliance_availability WHERE id = ?"
+                id_col_idx = 1
```

Ruff (0.15.0)

[error] Local variable `id_col_idx` is assigned to but never used

Remove assignment to unused variable `id_col_idx`

(F841)